kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -318,10 +315,6 @@
         message: 'View as grid',
         context: 'Label for a button used to view resources as a grid.',
       },
-      clearAll: {
```

Comment:
I think that we can remove the `results` and `moreThanXResults` strings above as well?